ballerina/admin.bal

```diff
@@ -0,0 +1,269 @@
+@display {label: "Azure Service Bus Administrator", iconPath: "icon.png"}
+public isolated client class Administrator {
+
+    final handle adminHandle;
+
+    #initialize the Azure Service Bus Admin client.
+    # Create an [Azure account](https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/) and
+    # obtain tokens following [this guide](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string).
+    # Configure the connection string to have the [required permission](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas).
+    #
+    # + connectionString - Azure Service Bus connection string
+    public isolated function init(@display {label: "Azure Service Bus connection string"} string connectionString) returns Error? {
+        handle|Error initResult = initializeAdmin(java:fromString(connectionString));
+        if (initResult is Error) {
+            return initResult;
+        }
+        self.adminHandle = initResult;
+    }
+
+    # Create a topic with the given name or name and options.
+    # 
+    # + topicName - Topic name
+    # + topicOptions - Topic options to create the topic.This should be a record of type CreateTopicOptions
+    # + return - Topic properties(Type of asb:TopicProperies) or error
+    @display {label: "Create Topic"}
+    isolated remote function createTopic(@display {label: "Topic"} string topicName, @display {label: "Topic Options"} *CreateTopicOptions topicOptions) returns TopicProperties|Error? =@java:Method {
+        'class: "org.ballerinax.asb.admin.Administrator"
+    } external;
+
+    # Get the topic with the given name.
+    # 
+    # + topicName - Topic name
+    # + return - Topic properties(Type of asb:TopicProperies) or error
+    @display {label: "Get Topic"}
+    isolated remote function getTopic(@display {label: "Topic"} string topicName) returns TopicProperties|Error? =@java:Method {
+        'class: "org.ballerinax.asb.admin.Administrator"
+    } external;
+
+    # Update the topic with the given options.
+    # 
+    # + topicName - Topic name
+    # + topicOptions - Topic options to update the topic.This should be a record of type UpdateTopicOptions
+    # + return - Topic properties(Type of asb:TopicProperies) or error
+    @display {label: "Update Topics"}
+    isolated remote function updateTopic(@display {label: "Topic"} string topicName, @display {label: "Update Topic Options"} *UpdateTopicOptions topicOptions) returns TopicProperties|Error? =@java:Method {
+        'class: "org.ballerinax.asb.admin.Administrator"
+    } external;
+
+    # List the topics.
+    # 
+    # + return - Topic list(Type of asb:TopicList) or error
+    @display {label: "List Topics"}
+    isolated remote function listTopics() returns TopicList|Error? =@java:Method {
+        'class: "org.ballerinax.asb.admin.Administrator"
+    } external;
+
+    # Delete the topic with the given name.
+    # w
```

Comment:
Hope this is a mistake

Reply:
Addressed in [af47f42](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/af47f42e025c870c173c07d46a984ed796dd7ab1).